geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```diff
@@ -312,63 +299,47 @@ public void testTombstoneExpirationRace() {
     VM vm1 = VM.getVM(1);
 
     final String name = this.getUniqueName() + "-CC";
-    SerializableRunnable createRegion = new SerializableRunnable("Create Region") {
+    SerializableRunnable createRegion = new SerializableRunnable() {
       @Override
       public void run() {
-        try {
-          RegionFactory f = getCache().createRegionFactory(getRegionAttributes());
-          CCRegion = (LocalRegion) f.create(name);
-          CCRegion.put("cckey0", "ccvalue");
-          CCRegion.put("cckey0", "ccvalue"); // version number will end up at 4
-        } catch (CacheException ex) {
-          ex.printStackTrace();
-          fail("While creating region ", ex);
-        }
+        RegionFactory f = getCache().createRegionFactory(getRegionAttributes());
+        CCRegion = (LocalRegion) f.create(name);
+        CCRegion.put("cckey0", "ccvalue");
+        CCRegion.put("cckey0", "ccvalue"); // version number will end up at 4;
       }
     };
 
-    vm0.invoke(createRegion);
-    vm1.invoke(createRegion);
-    vm1.invoke(new SerializableRunnable("Create local tombstone and adjust time") {
-      @Override
-      public void run() {
-        // make the entry for cckey0 a tombstone in this VM and set its modification time to be
-        // older
-        // than the tombstone GC interval. This means it could be in the process of being reaped by
-        // distributed-GC
-        RegionEntry entry = CCRegion.getRegionEntry("cckey0");
-        VersionTag tag = entry.getVersionStamp().asVersionTag();
-        assertThat(tag.getEntryVersion()).isGreaterThan(1);
-        tag.setVersionTimeStamp(
-            System.currentTimeMillis() - TombstoneService.REPLICATE_TOMBSTONE_TIMEOUT - 1000);
-        entry.getVersionStamp().setVersionTimeStamp(tag.getVersionTimeStamp());
-        try {
-          entry.makeTombstone(CCRegion, tag);
-        } catch (RegionClearedException e) {
-
-          fail("region was mysteriously cleared during unit testing ", e);
-        }
+    vm0.invoke("Create Region", createRegion);
+    vm1.invoke("Create Region", createRegion);
+    vm1.invoke("Create local tombstone and adjust time", () -> {
+      // make the entry for cckey0 a tombstone in this VM and set its modification time to be
+      // older than the tombstone GC interval. This means it could be in the process of being
+      // reaped by distributed-GC
+      RegionEntry entry = CCRegion.getRegionEntry("cckey0");
+      VersionTag tag = entry.getVersionStamp().asVersionTag();
+      assertThat(tag.getEntryVersion()).isGreaterThan(1);
+      tag.setVersionTimeStamp(
+          System.currentTimeMillis() - TombstoneService.REPLICATE_TOMBSTONE_TIMEOUT - 1000);
+      entry.getVersionStamp().setVersionTimeStamp(tag.getVersionTimeStamp());
+      try {
+        entry.makeTombstone(CCRegion, tag);
+      } catch (RegionClearedException e) {
```

Comment:
Delete this `catch` block and change the methods to `throws Exception`.

Reply:
done